synapse/storage/schema/delta/56/drop_unused_event_tables.sql

```diff
@@ -0,0 +1,20 @@
+/* Copyright 2019 The Matrix.org Foundation C.I.C.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+-- these tables are never used.
+DROP TABLE IF EXISTS room_names;
```

Comment:
I use this quite a lot when spelunking in the database :sob:

Reply:
likewise - I use it to give a human name to rooms I'm dealing with

Reply:
well, if you think it's worth keeping, we can, but generally my feeling is that the fewer tables we have to keep up to date, and the less code we have to maintain to do so, the better.

Reply:
I suppose it's not much harder to get the name from the event_json table if absolutely needed.

Reply:
There's also the `room_state` table, which also contains the names (as well as topics, join_rules and history_visibilities) of all rooms for the room directory, so I really do think these are redundant.